metrotranscom/doorway
# feat: move ListingsCombined component into Doorway UIC to properly style it

# Pull Request Template

## Issue Overview

This PR addresses #133 


## Description

- Move ListingsCombined, ListingsMap and ListingList into doorway UIC rather than sites/public. This is needed to be able to properly restyle the ListingsCombined component in order to build the mobile hybrid list.
- Move all logic out of the above components so they don't need to call helper methods from sites/public. Now the props for these components have all the necessary data.

## How Can This Be Tested/Reviewed?

You should open the listings page on the public site and make sure it still works as intended with no styling changes.

## Checklist:


## Reviewer Notes:

Steps to review a PR:

- Read and understand the issue, and ensure the author has added QA notes
- Review the code itself from a style point of view
- Pull the changes down locally and test that the acceptance criteria is met
- Also review the acceptance criteria on the Netlify deploy preview (noting that these do not yet include any backend changes made in the PR)
- Either explicitly ask a clarifying question, request changes, or approve the PR if there are small remaining changes but the PR is otherwise good to go

## On Merge:

If you have one commit and message, squash. If you need each message to be applied, rebase and merge.
